lib/zebra.h

```diff
@@ -433,7 +433,8 @@ typedef enum {
 	SAFI_ENCAP = 4,
 	SAFI_EVPN = 5,
 	SAFI_LABELED_UNICAST = 6,
-	SAFI_MAX = 7
+	SAFI_FLOWSPEC = 7,
```

Comment:
I thought flowspec was an afi?

What am I missing here?

Reply:
from RFC5575, FLOWSPEC IPv4.
   In order to achieve this goal, we define an application-specific NLRI
   identifier (AFI=1, SAFI=133) along with specific semantic rules.